api/src/main/kotlin/edu/wgu/osmt/collection/CollectionController.kt

```diff
@@ -203,4 +205,28 @@ class CollectionController @Autowired constructor(
         val sizedIterable = auditLogRepository.findByTableAndId(CollectionTable.tableName, entityId = collection!!.id.value, offsetPageable = pageable)
         return ResponseEntity.status(200).body(sizedIterable.toList().map{it.toModel()})
     }
+
+    @GetMapping(RoutePaths.WORKSPACE_PATH, produces = [MediaType.APPLICATION_JSON_VALUE])
+    @ResponseBody
+    fun workspaceByOwner(
```

Comment:
This method name feels like a getter, but the method gets or creates a workspace. I wonder if that should be clarified in the method name. I can go both ways.